fe/fe-core/src/main/java/org/apache/doris/planner/ScanNode.java

```diff
@@ -62,8 +62,9 @@ abstract public class ScanNode extends PlanNode {
     protected String sortColumn = null;
     protected Analyzer analyzer;
 
-    public ScanNode(PlanNodeId id, TupleDescriptor desc, String planNodeName) {
+    public ScanNode(PlanNodeId id, TupleDescriptor desc, String planNodeName, NodeType nodeType) {
         super(id, desc.getId().asList(), planNodeName);
```

Comment:
```suggestion
        super(id, desc.getId().asList(), planNodeName, nodeType);
```

Reply:
In order to reduce file modifications, this PR only makes special modifications to SCAN_NODE, and will subsequently initialize Node as a parameter